docs/.vuepress/config.js

```diff
@@ -268,18 +249,7 @@ module.exports = {
             ]
           },
           {
-            title: 'Case studies',
```

Comment:
I would leave these. We relatively recently moved these up a level and I think this is still a good thing to bring visibility to